code/GF_LOGGA_VIT.svg

```diff
@@ -0,0 +1,45 @@
+	<path fill="#FFFFFF" d="M97.5,68.2c0,0,5.4-1.4,8,1.1c0,0,1,0.2,3,0c0,0,2.6-6.7,22.4-5.3l-1.8,3.1c0,0,6.4,0.3,7.6,2.3h2.8
+		l2.5-4.3c0,0,26.6,3.5,33.7-11.5c7.1-15-18-25.3-18-25.3s-6.2-0.2-8-1.8h-1.1l-0.1,1.3c0,0,14.7,4.8,17,17
+		c2.3,12.2-15.4,15.5-21.1,15.8c0,0,9.2-17,12.6-21.1l0.3-2.3l-10.1-1.6l-15,26.1l0,0c0,0-10.4-2.8-21.3,0.9c0,0,11.5-18.5,15.5-23
+		l0.2-3.5c0,0-7.2,0.7-10.4-1.3c0,0-13.3,21.7-18.7,31.8V68.2L97.5,68.2z"/>
+	<path fill="#FFFFFF" d="M88.8,96.6l-1.4,0.3c0,0,16.6,2,18.1-5.1c0.6-2.9-1.6-4.5-4.1-5.5c2.4-0.1,5.5-0.8,6.7-3
+		c3.7-7-12.1-6-12.1-6l-0.7,0.8c0,0,0.5,0,1.2,0l-9.1,18.7C87.4,96.9,88,96.7,88.8,96.6z M93,95.9h-0.1c1-1.6,3.3-5.5,5.5-9.1
+		c0.9,0.7,1.8,1.9,2,3.5C100.9,94.6,94.9,95.8,93,95.9z M105.1,81.2c0.6,2.9-3.8,3.8-5.7,4c1.6-2.6,3-4.8,3.8-5.8
+		C104.1,79.8,104.9,80.4,105.1,81.2z"/>
+	<path fill="#FFFFFF" d="M82.9,87.5c-0.1,3-0.8,6.5-2.7,9.6h4.6c0,0,6.1-8.4,2.5-19.7h-4.9c0,0-4.3,5.2-8.1,10.2
+		c-2.2,0.5-3.6,1-3.6,1l0.2,0.7c0.9-0.3,1.7-0.5,2.6-0.7c-1.4,1.8-2.7,3.6-3.6,4.9l-1,2.7l0.2,0.8h2.1c0,0,2.3-4.2,5.5-9
+		C79.1,87.7,81.3,87.5,82.9,87.5z M82.4,80.4c0,0,0.6,2.5,0.5,5.9c-2,0.1-3.8,0.3-5.4,0.6C79,84.7,80.7,82.4,82.4,80.4z"/>
+	<path fill="#FFFFFF" d="M128.1,82.1c0-6.2-12.5-4.7-12.5-4.7s-1.7,2.8-3.8,6.3l0,0v0.1c-2.4,3.9-5.2,8.8-6.9,11.8L105,97h6.2
+		c0,0,0.5-1.1,1.9-3.9c0,0,0.6-1.1,1.6-2.7c2.7,3.4,7,7.3,12.2,6.6v-0.7c0,0-5.7-1.7-9.2-8.8C121.4,87.7,128,87.3,128.1,82.1z
+		 M117.5,85.8c1.4-2.2,2.8-4.5,3.8-5.8c1.5,0.5,2.8,1.4,2.4,3.1C122.9,85.8,119.1,85.9,117.5,85.8z"/>
+	<path fill="#FFFFFF" d="M128.8,97h2.8c3.1-8.3,8.8-19,8.8-19l-5.3-1.4c-7.4,15.6-7.2,20.2-7.2,20.2L128.8,97z"/>
+	<path fill="#FFFFFF" d="M162.2,80.6l3.2,0.3c0,0,0.5-0.1,1.3-0.1c-2.2,3.4-7,10.6-9.4,14.9c-1.9-1.4-7-5.2-7.5-8.6
+		c0,0,8.5-2.7,12-7.1l-3.9-1.3l-9.3,5c0.9-1.4,1.7-2.5,2.3-3.2l0.1-2c0,0-4.1,0.4-6-0.8c0,0-7.6,12.5-10.8,18.3l0.1,1.1l6.4,0.1
+		c0,0,0.3-0.8,1.8-3.6c0,0,1.3-2.3,2.9-5c1,2.4,4.1,8,12,8.5l0,0c1.2,0.2,5,0.6,5.9,0.6c0,0,1.5-3.9,13.4-3c0,0,0.3-0.2,0.2-1.3
+		c0,0-6-1.6-12.3,0.5c0,0,1.7-2.7,3.7-5.8l9.3-1c0-0.5-0.2-0.8-0.2-0.8l-5.2-0.8l-2.6,0.5c1.5-2.2,2.9-4.4,3.9-5.7
+		c4-0.3,8.6-0.4,11.9-0.1l0.1-1l-3.6-1.1c0,0-1.2-0.2-1-0.9c0,0-7.9-0.3-19.1,2.7L162.2,80.6z"/>
+	<path fill="#FFFFFF" d="M178.7,97h5.2c0,0,7.3-10.8,10.8-15.6l1.6,15.6h4.9c0,0,10.8-19.5,11.6-21.7c0,0-1.7-0.5-5.8-0.3
+		c0,0-6,12.1-9.4,19.3l-1.4-15.5l0.1-1.4c0,0-4.4,0.4-6.3-0.8c0,0-8.1,13.2-11.4,19.4L178.7,97z"/>
+	<path fill="#FFFFFF" d="M72.3,84.8c0.3-0.4,0.2-1,0.2-1c-1.9-0.3-4.1-0.3-6.3-0.2c1.3-2.3,2.5-4.8,3.7-7.2
+		c6.8-1.4,12.7-1.6,12.7-1.6c0.1-0.9,0-1.2,0-1.2C69,73,55.7,78.8,55.7,78.8c1.6,1,1.3,2.2,1.3,2.2c2-1.2,4.3-2.2,6.7-3l-3.9,6.2
+		c-3,0.5-5.2,0.9-5.2,0.9l0.4,1.4c1.2-0.3,2.5-0.6,3.9-0.8L53,95.4l0.5,1.7h4.8c0,0,3.5-5.3,7.1-11.9C69.3,84.8,72.3,84.8,72.3,84.8
+		z"/>
+	<path fill="#FFFFFF" d="M75.9,109.2c0.9,1.8,1.4,2.6,1.4,2.6c46.5-12.2,139.7-1,139.7-1c0.1-1.1,0.2-1.6,0.2-1.6
+		C111.2,98.4,75.9,109.2,75.9,109.2z"/>
```

Comment:
Why is this so long? Would be great with an explenation!

Reply:
could it be done with only a link to something? 

here is mine I used:

```
 <!--here is the linksheet to get the plus and minus signs for the FAQ-->
    <link rel="stylesheet" href="https://code.ionicframework.com/ionicons/2.0.1/css/ionicons.min.css">)
```

Reply:
Same goes for this, this is the code for the logo for Guldfabriken.